commands/deploy.go

```diff
@@ -295,12 +334,11 @@ func deployImage(
 	registryAuth string,
 	deployFlags DeployFlags,
 	tlsInsecure bool,
+	readOnlyRootFilesystem bool,
 ) (int, error) {
 
 	var statusCode int
-	// default to a readable filesystem until we get more input about the expected behavior
-	// and if we want to add another flag for this case
-	readOnlyRootFilesystem := false
+	readOnlyRfs := deployFlags.readOnlyRootFilesystem || readOnlyRootFilesystem
```

Comment:
This will only allow you to override `readOnlyRootFilesystem` to be `true`, it will not allow `deployFlags` to override to `false`. Just something we should be conscious of and make sure we are ok with.

Reply:
This is up for debate, but :

1. `faas store deploy` doesn't have a `--readonly` flag
2. If a function is made RO, I don't believe someone should be able to override it to non-RO

I'm up for discussion, though. 😊👍